### PR TITLE
pd: reduce frequent redundant log output

### LINKED ISSUE #15041: pd: tikv print a lot of log when it disconnect with pd
## Development Task

inject ha-tikv-random-to-pdleader-network-partition-last-for-10m

```
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
[2023/06/28 18:58:26.437 +08:00] [WARN] [[pd.rs:1796](http://pd.rs:1796/)] ["failed to update max timestamp for region 105884: Pd(Other(\"[components/pd_client/src/[tso.rs:97](http://tso.rs:97/)]: TimestampRequest channel is closed\"))"]
```

### PR DESCRIPTION
**Pull request by lhy1024**
### What is changed and how it works?

Issue Number: Close #15041

What's Changed:

```commit-message
```

### Release note 

```release-note
None
```

### CODE REVIEW

File: components/raftstore-v2/src/worker/pd/misc.rs [hunk 1]
```diff
@@ -73,10 +75,13 @@ where
...
                     }
                     Err(e) => {
-                        warn!(
-                            logger,
-                            "failed to update max timestamp for region {}: {:?}", region_id, e
-                        );
+                        if last_log_ts.elapsed() > log_interval {
+                            warn!(
+                                logger,
+                                "failed to update max timestamp for region {}: {:?}", region_id, e
```
Comment (bufferflies) [inline components/raftstore-v2/src/worker/pd/misc.rs:81]:
```suggestion
                                "failed to update max timestamp for region {}: {:?}", 
                                "region_id" => region_id,
                                "error" => e
```

**Review by bufferflies — APPROVED** [review]
rest LGTM

File: components/raftstore-v2/src/worker/pd/misc.rs [hunk 2]
```diff
@@ -33,6 +33,8 @@ where
         let causal_ts_provider = self.causal_ts_provider.clone();
         let logger = self.logger.clone();
         let shutdown = self.shutdown.clone();
+        let log_interval = Duration::from_secs(1);
```
Comment (tonyxuqqi) [inline components/raftstore-v2/src/worker/pd/misc.rs:36]: I think 1 seconds may be still too much. Maybe 5 seconds?

**Review by tonyxuqqi — APPROVED** [review]

File: components/raftstore-v2/src/worker/pd/misc.rs (hunk 2 above)
Comment (lhy1024) [reply components/raftstore-v2/src/worker/pd/misc.rs:36]: ok

**Comment by lhy1024** [comment]
@tonyxuqqi  plz help me to merge it

**Review by disksing — APPROVED** [review]

**Pull request merged**